EventFilter/DTRawToDigi/plugins/DTuROSRawToDigi.cc

```diff
@@ -0,0 +1,467 @@
+//-------------------------------------------------
+//
+//   Class: DTuROSRawToDigi
+//
+//   L1 DT uROS Raw-to-Digi
+//
+//
+//
+//   Author :
+//   C. Heidemann - RWTH Aachen
+//   J. Troconiz  - UAM
+//
+//
+//--------------------------------------------------
+
+#include "DTuROSRawToDigi.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "CondFormats/DTObjects/interface/DTReadOutMapping.h"
+#include "CondFormats/DataRecord/interface/DTReadOutMappingRcd.h"
+
+#include "DataFormats/DTDigi/interface/DTuROSControlData.h"
+#include "DataFormats/MuonDetId/interface/DTWireId.h"
+#include "EventFilter/DTRawToDigi/interface/DTROChainCoding.h"
+#include "EventFilter/Utilities/interface/DTCRC.h"
+
+#include <iostream>
+
+
+DTuROSRawToDigi::DTuROSRawToDigi(const edm::ParameterSet& pset) {
+
+  produces<DTDigiCollection>();
+//  produces<std::vector<DTuROSFEDData>>("DTuROSWords");
```

Comment:
Is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant